docs/how-to-docs/anti-affinity.md

```diff
@@ -0,0 +1,45 @@
+## Affinity/Anti-affinity
+
+###  Overview
```

Comment:
great overview. Please add somewhere in there a link to the official k8s documentation